sdk/typescript/src/cryptography/signature.ts

```diff
@@ -64,6 +71,13 @@ export function parseSerializedSignature(serializedSignature: SerializedSignatur
 	const signatureScheme =
 		SIGNATURE_FLAG_TO_SCHEME[bytes[0] as keyof typeof SIGNATURE_FLAG_TO_SCHEME];
 
+	if (signatureScheme === 'MultiSig') {
+		return {
+			signatureScheme,
+			bytes,
```

Comment:
this may be a breaking change.  I want to be able to parse MultiSig signatures here without throwing.  The downside of this is that we can't extract a `signature` and `publicKey` here, so those fields become optional in the response (unless you discriminate on signatureSchema)